agent/src/cli/cody-bench/EvaluationDocument.ts

```diff
@@ -299,6 +300,7 @@ export const headerItems: ObjectHeaderItem[] = [
     { id: 'editDiff', title: 'EDIT_DIFF' },
     { id: 'chatReply', title: 'CHAT_REPLY' },
     { id: 'chatQuestion', title: 'CHAT_QUESTION' },
+    { id: 'questionClass', title: 'QUESTION_CLASS' },
```

Comment:
We may want to have the name of the "task" as a prefix in the header
```suggestion
    { id: 'questionClass', title: 'CHAT_QUESTION_CLASS' },
```

Reply:
I liked this more generic name, since we can use it to categorize other tasks as well!